scijava/scijava-ops-engine/src/main/java/org/scijava/ops/engine/matcher/simplify/SimplifiedOpDescriptionGenerator.java

```diff
@@ -0,0 +1,133 @@
+/*-
+ * #%L
+ * SciJava Operations Engine: a framework for reusable algorithms.
+ * %%
+ * Copyright (C) 2016 - 2023 SciJava developers.
+ * %%
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are met:
+ *
+ * 1. Redistributions of source code must retain the above copyright notice,
+ *    this list of conditions and the following disclaimer.
+ * 2. Redistributions in binary form must reproduce the above copyright notice,
+ *    this list of conditions and the following disclaimer in the documentation
+ *    and/or other materials provided with the distribution.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+ * AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+ * IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+ * ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDERS OR CONTRIBUTORS BE
+ * LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+ * CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+ * SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+ * INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+ * CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+ * ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+ * POSSIBILITY OF SUCH DAMAGE.
+ * #L%
+ */
+
+package org.scijava.ops.engine.matcher.simplify;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import org.scijava.ops.api.OpEnvironment;
+import org.scijava.ops.api.OpInfo;
+import org.scijava.ops.api.OpRequest;
+import org.scijava.ops.engine.OpDescriptionGenerator;
+import org.scijava.ops.engine.util.Infos;
+import org.scijava.priority.Priority;
+
+/**
+ * An {@link OpDescriptionGenerator} implementation which makes use of
+ * {@link SimplifiedOpInfo}s to provide "simple" descriptions.
+ * 
+ * @author Gabriel Selzer
+ */
+public class SimplifiedOpDescriptionGenerator implements
+	OpDescriptionGenerator
+{
+
+	@Override
+	public String simpleDescriptions(OpEnvironment env, OpRequest req) {
+		String name = req.getName();
+		if (name == null) {
+			return allNamespaces(env);
+		}
+		if (env.infos(name).isEmpty()) {
+			return allNamespaces(env, name);
+		}
+		var infos = SimplificationMatchingRoutine.getSimpleInfos(env, req.getName());
```

Comment:
Man, I really hate this line. I dislike that we have a separate data structure storing `OpInfo`s that's not in the `DefaultOpEnvironment`, *and* I dislike how the way this is written will ignore all Ops that can't be simplified (which in this case is any `Computer`/`Inplace` whose output can't be copied).

Reply:
>I dislike how the way this is written will ignore all Ops that can't be simplified (which in this case is any Computer/Inplace whose output can't be copied).

@gselzer this sounds particularly concerning.. what're the implications here? Does that mean that these ops won't show up in an `ops.help` call?

Reply:
> > I dislike how the way this is written will ignore all Ops that can't be simplified (which in this case is any Computer/Inplace whose output can't be copied).
> 
> @gselzer this sounds particularly concerning.. what're the implications here? Does that mean that these ops won't show up in an `ops.help` call?

@hinerm yes, if an Op cannot be simplified then it will not show up in an `OpEnvironment.help` call (but it will still show up in an `OpEnvironment.helpVerbose` call)

Reply:
Fixed (albeit HACKy) with 6bcee354